src/services/accounts/AccountsStakingInfoService.ts

```diff
@@ -66,6 +67,64 @@ export class AccountsStakingInfoService extends AbstractService {
 			);
 		}
 
+		let claimedRewardsEras: u32[] = [];
+		let claimedRewards: IEraStatus[] = [];
+
+		if ((stakingLedger as unknown as IStakingLedger)?.lastReward) {
+			const e = (stakingLedger as unknown as IStakingLedger)?.lastReward?.unwrap().toNumber();
```

Comment:
Is it possible to unwrap a `None` value here?

Reply:
Very good point! I added a check in this [commit](https://github.com/paritytech/substrate-api-sidecar/pull/1445/commits/753f3eefde99a1fa750a972ba395ee53017131e5)